qjs-wasi/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ add_definitions(
     -DCONFIG_VERSION="2019-07-09"
 )
 
-add_executable(qjs.wasm
+add_executable(qjs
```

Comment:
Note, the `qjs.wasm` will be automatically generated (and `qjs` will be just an executable wrapper to execute the wasm file with a standalone WebAssembly runtime)